src/js/profile/mobile/widget/Calendar.js

```diff
@@ -40,19 +44,23 @@
 				events = ns.event,
 				days = ["MON", "TUE", "WED", "THU", "FRI", "SAT", "SUN"],
 				fullNameMonth = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"],
-				today = new Date(),
-				todayYear = today.getFullYear(),
-				todayMonth = today.getMonth() + 1,
-				defaultToday = today.getUTCDate(),
+				dateData = new Date(),
+				todayYear = dateData.getFullYear(),
+				todayMonth = dateData.getMonth() + 1,
+				defaultToday = dateData.getUTCDate(),
+				loadTodayFlag = true,
+				activeMonth,
+				fixMonth,
 				switchElement,
+				arrowElement,
 				selectDay,
 
 				Calendar = function () {
 					this.options = utilsObject.merge({}, defaultOptions);
 				},
```

Comment:
add `this._ui = {};`

For store important HTML elements of widget, eg. `calendarView`

Reply:
ok.

done.